content/en/blog/identity-hack-panel-discussion/index.md

```diff
@@ -0,0 +1,119 @@
+---
+date: 2023-05-16
+title: "Digital Identity Hack Part 2 – Panel Discussion on Blockchain Projects"
+linkTitle: "Digital Identity Hack Part 2 – Panel Discussion on Blockchain Projects"
+description: "In this blog, we discuss about the Digital Identity Hack event's panel discussion, which focused on exploring development of blockchain projects in various companies."
+author: Pekka Kaipio
+resources:
+  - src: "**.{png,jpg}"
+    title: "Image #:counter"
+---
+The emergence of **self-sovereign identity** (SSI) offers a revolutionary
+opportunity for innovation and growth in the financial world, with the potential
+to transform digital financial services, streamline transactions, and enable
```

Comment:
with the potential... -> potentially transforming digital financial services, streamlining transactions, and enabling seamless interaction.